python/cugraph/__init__.py

```diff
@@ -1,7 +1,25 @@
 # Copyright (c) 2018, NVIDIA CORPORATION.
-# Versioneer
-import cugraph
 
+from cugraph.community import (
```

Comment:
Thank you very much for pointing this out and other missing copyright headers. There was another email thread discussing what to include in this PR, and I was asked to limit updates in a single PR but I will use your feedback as an excuse to bring this additional updates to this PR :-)

Reply:
@dantegd @afender @BradReesWork

We have two kinds of license headers:
a short one
```
# Copyright (c) 2018, NVIDIA CORPORATION.
```
and a longer one
```
# Copyright (c) 2019, NVIDIA CORPORATION.
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```
Which one should I use? And the year should be 2019 if the file is newly created in 2019 and 2018-2019 if the file is first created in 2018, is this correct?

Reply:
> Which one should I use? 

The longer one (Apache 2.0).

> And the year should be 2019 if the file is newly created in 2019 and 2018-2019 if the file is first created in 2018, is this correct?

AFAIK, what really matters is to show the date of the first publication. It indicates how far back the copyright claim is made. We can add other dates (in addition to the first one) when adding substantial changes to the code This may help increasing copyright duration (which is already pretty long).

Reply:
Added license headers.